src/main.c

```diff
@@ -761,6 +761,33 @@ static NETX_CONSOLEAPP_RESULT_T opMode_spiMacroPlayer(tFlasherInputParameter *pt
 }
 
 
+/* ------------------------------------- */
+
+static NETX_CONSOLEAPP_RESULT_T opMode_identify(tFlasherInputParameter *ptAppParams, NETX_CONSOLEAPP_PARAMETER_T *ptConsoleParams)
+{
+	NETX_CONSOLEAPP_RESULT_T tResult;
+	(void) ptConsoleParams; /* unused */
+	(void) ptAppParams; /* unused */
+
+	const unsigned int uDelayTimeMs = 125U;
+	const unsigned int uBlinkDurationMs = 5000U;
+
+	unsigned int blinks = 0U;
+
+	while(blinks < (uBlinkDurationMs/uDelayTimeMs)){
+		systime_delay_ms(uDelayTimeMs);
+		RDYRUN_T ledColor = blinks%8<4?RDYRUN_YELLOW:RDYRUN_GREEN;
+		rdy_run_setLEDs((blinks%2==1?ledColor:RDYRUN_OFF));
+		blinks++;
+	}
+
+	rdy_run_setLEDs(RDYRUN_OFF);
```

Comment:
This is not necessary, as the main program will set the LEDs to display the return code.